lib/miq_environment.rb

```diff
@@ -83,11 +83,10 @@ def self.rails_command
 
     def self.supports_command?(cmd)
       return false unless EVM_KNOWN_COMMANDS.include?(cmd)
-      require "runcmd"
 
       begin
         # If 'which apachectl' returns non-zero, it wasn't found
-        MiqUtil.runcmd(which, :params => [[cmd]])
+        AwesomeSpawn.run!(which, :params => [cmd])
```

Comment:
This can arguably be changed to `AwesomeSpawn.run(which, :params => [cmd]).success?`, which would avoid the exception handling overhead, but that can be a separate PR.